sdk/core/perf/README.md

```diff
@@ -152,7 +152,7 @@ int main(int argc, char** argv)
            }}};
 
   // Call the `Run` method with a context, the tests and the application arguments to launch the program.
-  Azure::Perf::Program::Run(Azure::Core::Context::GetApplicationContext(), tests, argc, argv);
+  Azure::Perf::Program::Run(Azure::Core::Context::ApplicationContext, tests, argc, argv);
```

Comment:
Any other readmes or particular doxygen docs that need updating?

Reply:
I haven't seen any - to my best understanding and effort, no more doxy comments need to be updated